tasks_week3_odelya_yaakowich/include/Dog.h

```diff
@@ -0,0 +1,35 @@
+/*********************
+Name: Odelya Yaakowich
+ID: 207346784
+**********************/
+
+
+#pragma once
+#include "Animal.h"
+#include <ctime>
+#include <cstdlib>
+#define ROWS 20
+#define COLS 40
+
+class Dog : public Animal {
+    bool turn = false;
+    int direction = 0;
```

Comment:
direction is common variable between classes, it should be in the animal class, under the protected section